components/collapsible/accordion/src/index.js

```diff
@@ -0,0 +1,54 @@
+import React, {Component, PropTypes} from 'react'
+import CollapsibleBasic from '@schibstedspain/sui-collapsible-basic'
+
+class CollapsibleAccordion extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {items: props.items}
+    this._handleClick = this._handleClick.bind(this)
+    this._collapseItems = this._collapseItems.bind(this)
+  }
+
+  _handleClick (id) {
+    return (collapsed) =>
+      this._collapseItems(collapsed, id)
+  }
+
+  _collapseItems (collapsed, id) {
```

Comment:
For performance, You could remember was the las id and then you wont have to iterate on every render.
And, please implement the shouldComponentUpdate hook.